org.eclipse.xtext.common.types.tests/src-gen/org/eclipse/xtext/common/types/xtext/ui/parser/antlr/internal/InternalContentAssistTestLanguageLexer.java

```diff
@@ -835,67 +835,70 @@ public void mTokens() throws RecognitionException {
         "\7\uffff\1\7\1\uffff\1\11\1\12\3\uffff\1\16\1\17\1\uffff\1\11\4\uffff\1\6\1\10\1\7\1\12\1\13\1\14\1\15\1\16\25\uffff\1\2\2\uffff\1\5\1\1\1\3\1\uffff\1\4";
     static final String DFA12_specialS =
         "\1\2\12\uffff\1\0\1\1\56\uffff}>";
-    static final String[] DFA12_transitionS = {
-            "\11\17\2\16\2\17\1\16\22\17\1\16\1\17\1\13\1\17\1\7\2\17\1\14\6\17\1\6\1\15\12\12\7\17\32\11\3\17\1\10\1\11\1\17\2\11\1\2\1\1\2\11\1\4\1\11\1\5\11\11\1\3\7\11\uff85\17",
-            "\1\20",
-            "\1\22",
-            "\1\23",
-            "\1\24",
-            "\1\25",
-            "\1\26",
-            "",
-            "\32\21\4\uffff\1\21\1\uffff\32\21",
-            "",
-            "",
-            "\0\32",
-            "\0\32",
-            "\1\33\4\uffff\1\34",
-            "",
-            "",
-            "\1\36",
-            "",
-            "\1\37",
-            "\1\40",
-            "\1\41",
-            "\1\42",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "\1\43",
-            "\1\44",
-            "\1\45",
-            "\1\46",
-            "\1\47",
-            "\1\50",
-            "\1\51",
-            "\1\52",
-            "\1\53",
-            "\1\54",
-            "\1\55",
-            "\1\56",
-            "\1\57",
-            "\1\60",
-            "\1\61",
-            "\1\62",
-            "\12\21\7\uffff\32\21\4\uffff\1\21\1\uffff\32\21",
-            "\1\64",
-            "\1\65",
-            "\12\21\7\uffff\32\21\4\uffff\1\21\1\uffff\32\21",
-            "\12\21\7\uffff\32\21\4\uffff\1\21\1\uffff\32\21",
-            "",
-            "\12\21\7\uffff\32\21\4\uffff\1\21\1\uffff\32\21",
```

Comment:
@szarnekow what is the actual diff here?

Reply:
Indentation because it's now nested in a static inner class. See first two lines of the diff block.